tests/unit/build_charms/conftest.py

```diff
@@ -0,0 +1,16 @@
+"""Ensure that path is included in PYTHONPATH before tests are run."""
+import importlib
+import sys
+import pytest
+
+
+@pytest.fixture(scope="package")
+def charms():
```

Comment:
:heart:

Reply:
this "sneak in" was part of adding new unit tests.  because the build python are scattered around, they aren't in python packages -- they're just bare python files.  This means the tests have to open/close scope on the python path.  "charms.py" in one directory isn't necessarily the same module as "charms.py" in other path.

Reply:
Makes sense, thanks!